<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CranioViewer</class>
 <widget class="QMainWindow" name="CranioViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1451</width>
    <height>1067</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CranioViewer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="m_CentralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="1,1">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_YZ">
        <item>
         <widget class="QVTKWidget" name="m_YZViewer"/>
        </item>
        <item>
         <widget class="QSlider" name="m_YZSlider">
          <property name="value">
           <number>49</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QVTKWidget" name="m_3DViewer"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_XZ">
        <item>
         <widget class="QVTKWidget" name="m_XZViewer"/>
        </item>
        <item>
         <widget class="QSlider" name="m_XZSlider">
          <property name="value">
           <number>49</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_XY">
        <item>
         <widget class="QVTKWidget" name="m_XYViewer"/>
        </item>
        <item>
         <widget class="QSlider" name="m_XYSlider">
          <property name="value">
           <number>49</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="m_StatusBar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1451</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_MenuFile">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="m_OpenAction"/>
    <addaction name="m_LoadTemplateMesh"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionTestITK"/>
    <addaction name="actionSaveMesh"/>
    <addaction name="actionSaveImg"/>
    <addaction name="actionControlPanel"/>
    <addaction name="actionMorphingViewer"/>
   </widget>
   <addaction name="m_MenuFile"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QDockWidget" name="dockControlPanel">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Transfrom</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>5</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="m_LabelTranslation">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Translation</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="m_LabelLR">
              <property name="text">
               <string>LR</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="m_SliderTransLR">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>40000</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>20000</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m_SpinBoxTransLR">
              <property name="minimum">
               <double>-200.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="m_LabelPA">
              <property name="text">
               <string>PA</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="m_SliderTransPA">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>40000</number>
              </property>
              <property name="value">
               <number>20000</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m_SpinBoxTransPA">
              <property name="minimum">
               <double>-200.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.050000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="m_LabelIS">
              <property name="text">
               <string>IS</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="m_SliderTransIS">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>40000</number>
              </property>
              <property name="value">
               <number>20000</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m_SpinBoxTransIS">
              <property name="minimum">
               <double>-200.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.050000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="m_LabelTransMax">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m_SpinBoxTransMax">
              <property name="minimum">
               <double>-2000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1999.990000000000009</double>
              </property>
              <property name="value">
               <double>200.000000000000000</double>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="m_LabelTransMin">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m_SpinBoxTransMin">
              <property name="minimum">
               <double>-2000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1999.990000000000009</double>
              </property>
              <property name="value">
               <double>-200.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="m_LabelRotation">
            <property name="text">
             <string>Rotation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_SliderRotationLR">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_SliderRotationPA">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_SliderRotationIS">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <property name="spacing">
           <number>5</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="m_LabelScale">
            <property name="text">
             <string>Scale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_SliderScale">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Setting</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
          <item>
           <widget class="QCheckBox" name="m_CheckBoxShowSkull">
            <property name="text">
             <string>Show Skull</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="m_LabelChangeRepresent">
              <property name="text">
               <string>Disp Type</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_ComboBoxChangeRepresent">
              <item>
               <property name="text">
                <string>Surface</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wireframe</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Points</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
          <item>
           <widget class="QCheckBox" name="m_CheckBoxShowCTData">
            <property name="text">
             <string>Show CT Data</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_PushButtonClrCTData">
            <property name="text">
             <string>Clear CT Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="m_CheckBoxShowIntersection">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Show Intersection</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Volume Disp</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QCheckBox" name="m_showVolumeCheckBox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="m_ISOValSlider">
              <property name="minimum">
               <number>-2000</number>
              </property>
              <property name="maximum">
               <number>2000</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_ISOValSpinBox">
              <property name="minimum">
               <number>-2000</number>
              </property>
              <property name="maximum">
               <number>2000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_ISOWidthSpinBox">
              <property name="maximum">
               <number>2000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Registration</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="m_PushButtonICP">
          <property name="text">
           <string>Rigid</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QSpinBox" name="spinBoxOutIter">
            <property name="value">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxInIter">
            <property name="value">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxGradStep">
            <property name="value">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="m_PushButtonAddCrsp">
          <property name="text">
           <string>Manually Set Correspondences</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="m_PushButtonNonRigid">
          <property name="text">
           <string>Non Rigid</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="m_PushButtonNonRigidIter">
          <property name="text">
           <string>Non Rigit Iter</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxLamdArap">
            <property name="value">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxLamdUserCrsp">
            <property name="value">
             <double>25.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>300</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="m_OpenAction">
   <property name="text">
    <string>Load NIFTI File</string>
   </property>
  </action>
  <action name="m_LoadTemplateMesh">
   <property name="text">
    <string>Load Template Mesh</string>
   </property>
  </action>
  <action name="m_ICPPoint2Point">
   <property name="text">
    <string>ICP Point2Point</string>
   </property>
  </action>
  <action name="m_ICPPoint2Plane">
   <property name="text">
    <string>ICP Point2Plane</string>
   </property>
  </action>
  <action name="actionTestITK">
   <property name="text">
    <string>TestITK</string>
   </property>
  </action>
  <action name="actionSaveMesh">
   <property name="text">
    <string>SaveMesh</string>
   </property>
  </action>
  <action name="actionSaveImg">
   <property name="text">
    <string>SaveImg</string>
   </property>
  </action>
  <action name="actionControlPanel">
   <property name="text">
    <string>Control Panel</string>
   </property>
  </action>
  <action name="actionMorphingViewer">
   <property name="text">
    <string>Morphing</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
